templates/zerver/integrations/google-calendar.md

```diff
@@ -1,72 +1,64 @@
-Get Google Calendar reminders in Zulip! This is a great way to see
+# Zulip Google Calendar integration
+
+Get Google Calendar reminders in your Zulip DMs! This is a great way to see
 your reminders directly in your Zulip feed.
 
-1.  {!download-python-bindings.md!}
+!!! warn ""
 
     This bot should be set up on a trusted machine, because your API
     key is visible to local users through the command line or config
     file.
 
-1.  Next, follow the instructions for **Step 1** at
-    [this link](https://developers.google.com/google-apps/calendar/quickstart/python)
-    to get a `client_secret` file. Save this file as `client_secret.json`
-    to your `~/` directory.
+{start_tabs}
 
-1.  Next, install the latest Google API Client for Python by following the
-    instructions on the
-    [Google website](https://developers.google.com/api-client-library/python/start/installation).
+1.  Download and install our
```

Comment:
Previously, the macro `{!download-python-bindings.md!}` was used here, but it mentions "server" which can be confusing because unlike other integrations that can be added to entire organizations, this one's geared for individual users. So, pasted the macro content directly with the tiny tweak to remove the "server" mention.